packages/core/src/config/pixel-format.ts

```diff
@@ -31,12 +31,15 @@ export const validateSelectedPixelFormatAndCodecCombination = (
 	pixelFormat: PixelFormat,
 	codec: Codec
 ) => {
+	if (!validOptions.includes(pixelFormat)) {
+		throw new TypeError(`Value ${pixelFormat} is not valid as a pixel format.`);
```

Comment:
Add the same check as it was done in setPixelFormat() to ensure, that the passed pixelformat is valid. Currently an invalid pixelformat would go as a valid pixelformat/codec combination.